docs/references/javascript/sign-in/sign-in.mdx

```diff
@@ -53,7 +53,7 @@ function create(params: SignInCreateParams): Promise<SignIn>;
+| `strategy?` | `string` | Select the first factor verification strategy. The `strategy` value depends on the object's `identifier` value. Each authentication identifier supports different verification strategies.<br/>Possible `strategy` values are: <ul><li>`password`: The verification will attempt to be completed using the user's password.</li><li>`email_link`: User will receive an email magic link via email. The `email_address_id` parameter can also be specified to select one of the user's known email addresses.</li><li>`email_code`: User will receive a one-time authentication code via email. At least one email address should be on file for the user. The `email_address_id` parameter can also be specified to select one of the user's known email addresses.</li><li>`phone_code`: User will receive a one-time authentication code in their phone, via SMS. At least one phone number should be on file for the user. The `phone_number_id` parameter can also be specified to select which of the user's known phone numbers the SMS will go to.</li><li>`web3_metamask_signature`: The verification will attempt to be completed using the user's web3 wallet public address. The `web3_wallet_id` parameter can also be specified to select which of the user's known web3 wallets will be used. Currently Clerk supports [Metamask](https://metamask.io/).</li><li>`oauth_<provider>`: The user will be authenticated with their social sign-in account. See available [OAuth providers](/docs/references/javascript/types/oauth#o-auth-provider).</li><li>`ticket`: The user will be authenticated via the ticket *or token* generated from the Backend API.</li><li>`google_one_tap`: The user will be authenticated with Google One Tap UI. It's recommended to use [`authenticateWithGoogleOneTap()`](/docs/components/authentication/google-one-tap#authenticate-with-google-one-tap) instead, as it will also set the user's current session as active.</li></ul> |
```

Comment:
```suggestion
| `strategy?` | `string` | Select the first factor verification strategy. The `strategy` value depends on the object's `identifier` value. Each authentication identifier supports different verification strategies.<br/>Possible `strategy` values are: <ul><li>`password`: The verification will attempt to be completed using the user's password.</li><li>`email_link`: User will receive an email magic link via email. The `email_address_id` parameter can also be specified to select one of the user's known email addresses.</li><li>`email_code`: User will receive a one-time authentication code via email. At least one email address should be on file for the user. The `email_address_id` parameter can also be specified to select one of the user's known email addresses.</li><li>`phone_code`: User will receive a one-time authentication code in their phone, via SMS. At least one phone number should be on file for the user. The `phone_number_id` parameter can also be specified to select which of the user's known phone numbers the SMS will go to.</li><li>`web3_metamask_signature`: The verification will attempt to be completed using the user's web3 wallet public address. The `web3_wallet_id` parameter can also be specified to select which of the user's known web3 wallets will be used. Currently Clerk supports [Metamask](https://metamask.io/).</li><li>`oauth_<provider>`: The user will be authenticated with their social sign-in account. See available [OAuth providers](/docs/references/javascript/types/oauth#o-auth-provider).</li><li>`ticket`: The user will be authenticated via the ticket *or token* generated from the Backend API.</li><li>`google_one_tap`: The user will be authenticated with the Google One Tap UI. It's recommended to use [`authenticateWithGoogleOneTap()`](/docs/components/authentication/google-one-tap#authenticate-with-google-one-tap) instead, as it will also set the user's current session as active for you.</li></ul> |
```